Tst/UnitTests/RegressionTests.cs

```diff
@@ -81,7 +81,7 @@ public void TestTemp()
     public class PrtSharpRegressionTests
     {
         private static IEnumerable<TestCaseData> RegressionTestSuite =>
-            TestCaseLoader.FindTestCasesInDirectory(Constants.TestDirectory, new[] {"Prt", "PrtSharp" });
+            TestCaseLoader.FindTestCasesInDirectory(Constants.TestDirectory, new[] {"DynamicError", "DynamicErrorPrtSharp", "Correct" });
```

Comment:
Is DynamicErrorPrtSharp a separate folder?

Reply:
Yes, there are a few cases that seem to be written for only C# or C runtime. I modified the parsing logic to and renamed few folders to address this need. 

Also, static errors are platform independent, so they are separate from either PrtRegressions or PrtSharpRegressions

Reply:
Can you please point out these cases? I would like to look at these and then fix them in some ways to make it uniform.

Reply:
Yes, there are only 2 such cases which are located in this folder:
Feature2Stmts/DynamicErrorPrtSharp/

Reply:
Thanks, Eric, for identifying these (even the unused) cases.

Can I request you to add DynamicError* testcases to DynamicError and Correct* to Correct.

Please let me know if they dont pass for Prt or PrtSharp.

Reply:
Yes, actually, all the *Unused cases are not passing which is why they are included in a separate folder. Current set of StaticError, DynamicError and Correct is the maximum passing set

Reply:
Thanks Eric, for all this experiments. Can I ask you to do the following final change:
Create the appropriate folders such that the *Unused cases also pass and are included in the regressions.
I think most of them will pass the *PrtSharp.